<launch>
<node name = "turtlesim" pkg = "turtlesim" type = "turtlesim_node" output = "screen"/>
<node name = "turtlesim_cleaner_node" pkg = "hello_world" type = "motion_cleaner.py" output = "screen"/>
</launch>